reference/SimpleCalculator/SimpleCalculator.Shared/Keyboard/KeyValues.cs

```diff
@@ -7,7 +7,9 @@ namespace SimpleCalculator.Keyboard;
 
 public static class KeyValues
 {
-    public static Dictionary<string, KeyInput> Keys = new()
+    public static IReadOnlyDictionary<string, KeyInput> Keys => _keys;
+
+    private static Dictionary<string, KeyInput> _keys = new()
```

Comment:
can you ditch the private backing field... just make the public property readonly